File: module3/10.JSP&TSTL/BaiTap/B1.DanhSachKhachHang/src/model/Customer.java
package model;

public class Customer {
    private String name;
    private String date_of_birth;
    private String address;
    private String imageURL;

    public Customer() {
    }

    public Customer(String name, String date_of_birth, String address, String imageURL) {
        this.name = name;
        this.date_of_birth = date_of_birth;
        this.address = address;
        this.imageURL = imageURL;
    }

    public String getName() {
        return name;
    }

    public void setName(String name) {
        this.name = name;
    }

    public String getDate_of_birth() {
        return date_of_birth;
    }

    public void setDate_of_birth(String date_of_birth) {
        this.date_of_birth = date_of_birth;
    }

    public String getAddress() {
        return address;
    }

    public void setAddress(String address) {
        this.address = address;
    }

    public String getImageURL() {
        return imageURL;
    }

    public void setImageURL(String imageURL) {
        this.imageURL = imageURL;
    }
}

File: modul1/CaseStudy/CaseStusy6/Task1/Customer.js
class Customer {
    constructor() {
    }
        setName(name) {
            this.name = name;
        };
        getName() {
            return this.name
        };
        setCmnd(cmnd) {
            this.cmnd = cmnd;
        };
        getCmnd() {
            return this.cmnd
        };
        setBirthDay(birthDay) {
            this.birthDay = birthDay;
        };
        getBirthDay() {
            return this.birthDay
        };
        setEmail(email) {
            this.email = email;
        };
        getEmail() {
            return this.email
        };
       setAddress(address) {
            this.address = address;
        };
       getAddress() {
            return this.address
        };
       setTypeCustomer(typeCustomer) {
            this.typeCustomer = typeCustomer;
        };
        getTypeCustomer() {
            return this.typeCustomer
        };
       setDiscount(discount) {
            this.discount = discount;
        };
       getDiscount() {
            return this.discount
        };
       setPeople(people) {
            this.people = people;
        };
        getPeople() {
            return this.people
        };
        setRentDays(rentDays) {
            this.rentDays = rentDays;
        };
        getRentDays() {
            return this.rentDays
        };
        setTypeService(typeService) {
            this.typeService = typeService;
        };
        getTypeService() {
            return this.typeService
        };
        setTypeRoom(typeRoom) {
            this.typeRoom = typeRoom;
        };
        getTypeRoom() {
            return this.typeRoom
        };
        totaPays() {
            let money = 0;
            if (this.getTypeService() === "Villa") {
                money = 500;
            } else if (this.getTypeService() === "House") {
                money = 300;
            } else if (this.getTypeService() === "Room") {
                money = 200;
            }
            return this.getRentDays() * money * (1 - this.getDiscount() / 100);
        }
    }
